CHANGELOG.md

```diff
@@ -25,6 +26,17 @@
  - [0.0.5](#005---20180602)
  - [0.0.4 and prior](#004-and-prior)
 
+## [1.1.1] - 2021/01/07
```

Comment:
We reliably forget to add these because they don't look like MD links :)

This needs a compare line at the bottom similar to https://github.com/Kong/kubernetes-ingress-controller/blame/master/CHANGELOG.md#L945